src/Mapbender/PrintBundle/Resources/public/element/batchprintclient.js

```diff
@@ -0,0 +1,1494 @@
+            };
+            
+            $mapElement.on('click', this.mouseClickHandler);
+            
+            // Mouse move handler - update feature position
+            this.mouseMoveHandler = function(evt) {
+                if (!self.mouseFollowActive || !self.feature) {
+                    return;
+                }
+                
+                var pixel = map.getEventPixel(evt.originalEvent);
+                var coordinate = map.getCoordinateFromPixel(pixel);
+                
+                if (coordinate) {
+                    self._moveFeatureToCoordinate(coordinate);
+                }
+            };
+            
+            $mapElement.on('mousemove', this.mouseMoveHandler);
+        },
+        
+        /**
+         * Stop mouse-following behavior
+         */
+        _stopMouseFollow: function() {
+            if (!this.mouseFollowActive) {
+                return;
+            }
+            
+            this.mouseFollowActive = false;
+            
+            var map = this.map.getModel().olMap;
+            var $mapElement = $(map.getTargetElement());
+            
+            if (this.mouseMoveHandler) {
+                $mapElement.off('mousemove', this.mouseMoveHandler);
+                this.mouseMoveHandler = null;
+            }
+            
+            if (this.mouseClickHandler) {
+                $mapElement.off('click', this.mouseClickHandler);
+                this.mouseClickHandler = null;
+            }
+        },
+        
+        /**
+         * Move the current selection feature to a coordinate
+         */
+        _moveFeatureToCoordinate: function(coordinate) {
+            if (!this.feature) {
+                return;
+            }
+            
+            var geom = this.feature.getGeometry();
+            var currentCenter = this.map.getModel().getFeatureCenter(this.feature);
+            var dx = coordinate[0] - currentCenter[0];
+            var dy = coordinate[1] - currentCenter[1];
+            
+            geom.translate(dx, dy);
+        },
+        
+        /**
+         * Rotate the current selection feature by a bearing in degrees
+         * @param {number} bearingDegrees - Bearing in degrees from East (0 = East, 90 = North, -90 = South)
+         * @param {number|null} previousRotation - Previous frame's rotation in degrees (for continuity)
+         */
+        _rotateCurrentFeature: function(bearingDegrees, previousRotation) {
+            if (!this.feature) {
+                return;
+            }
+            
+            // The bearing represents the direction of the track (angle from East)
+            // We want to rotate the frame so this direction is parallel to one of its axes
+            
+            // Normalize bearing to -180 to 180 range
+            var normalizedBearing = bearingDegrees;
+            while (normalizedBearing > 180) normalizedBearing -= 360;
+            while (normalizedBearing <= -180) normalizedBearing += 360;
```

Comment:
This seems wildly inefficient. Why not use a modulo?


```suggestion
            const normalizedBearing = ((bearingDegrees + 180) % 360) - 180
```